CMakeLists.txt

```diff
@@ -684,6 +693,34 @@ function(AddApplication)
     set_property(TARGET ${A_Target} APPEND PROPERTY COMPILE_DEFINITIONS ${A_Definitions})
     set_target_properties(${A_Target} PROPERTIES OUTPUT_NAME "${A_ExecutableName}" PREFIX "" FOLDER "engine")
 
+    # Append Windows specific manifests.
+    if (WIN32)
+      set(WINDOWS_MANIFESTS_DIR ${ENGINE_DIR}/sys/manifests)
+      target_sources(${A_Target} PRIVATE
+          # App is DPI aware, so Windows doesn't try to scale UI.
+          ${WINDOWS_MANIFESTS_DIR}/dpi-aware.manifest
+          # App supports Windows Vista+, no quirks required.
+          ${WINDOWS_MANIFESTS_DIR}/supported-os.manifest
+          # Use visual styles for Windows UI controls (message boxes, buttons, etc.).
```

Comment:
The UI is all drawn with OpenGL, so this doesn't seem relevant. The only native control we have is the message box when the engine encounters a fatal error.

Reply:
Yep, but still message box can be styled as well.

Since this doesn't break anything, suggest to integrate manifest.

Reply:
The visual styles one in particular seems to significantly increase the complexity of the build system, so I don't think it is worth it.

Reply:
Good point. Removed.